{
  "affected": [
    {
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/npm/xavialonzo/MAL-2025-42118.json"
      },
      "package": {
        "ecosystem": "npm",
        "name": "xavialonzo",
        "purl": "pkg:npm/xavialonzo"
      },
      "ranges": [
        {
          "events": [
            {
              "introduced": "0"
            }
          ],
          "type": "SEMVER"
        }
      ]
    }
  ],
  "credits": [
    {
      "contact": [
        "actran@amazon.com"
      ],
      "name": "Amazon Inspector",
      "type": "FINDER"
    }
  ],
  "database_specific": {
    "malicious-packages-origins": null
  },
  "details": "The package xavialonzo was found to contain malicious code.",
  "id": "MAL-2025-42118",
  "modified": "2025-08-29T18:55:03Z",
  "published": "2025-08-29T18:55:03Z",
  "schema_version": "1.7.3",
  "summary": "Malicious code in xavialonzo (npm)"
}